### ISSUE TITLE
I can't get the website to use vertical alignment or to resize depending on how wide I make the browser.

### ISSUE THREAD

**Issue by maddycpierce** [maddycpierce/sections#1]
Hi Thomas, I can't figure out how to make the text align depending on the size of the browser. I've tried using vertical-align: middle and using max-width and width but I don't think that I'm doing things right. @thomasjbradley

**Comment by thomasjbradley** [comment]
We need to make a few small tweaks in the HTML:
- The `<img>` tag should be in a `<div>`.
- Then all the text and the button should be inside another `<div>`.

Those `<div>` tags are then put beside each other with `display`, `vertical-align` and `max-width`.

**Comment by maddycpierce** [comment]
I don't think that I did what you said correctly, because it still isn't working. Did I put the tags in the right places? @thomasjbradley

**Comment by maddycpierce** [comment]
Oh wait I got it!! Thank you! @thomasjbradley

**Comment by maddycpierce** [comment]
I am now getting these issues and I don't know how to correct them. Each time that I remove the h2 element then it tells me that it's required, and each time I put in a p element into sections then it tells me that p can't be an element of h2 in this situation. What should I do please? Also, thank you for your time helping me! @thomasjbradley  
![sections-screenshot-1](https://cloud.githubusercontent.com/assets/22149579/19449758/e466e970-9474-11e6-8b96-0f1ba23b108e.jpg)

**Comment by thomasjbradley** [comment]
Change the `<h1>` to an `<h2>`, then change the `<h2>` to a `<p>`.

---

For the font, when you chose it on Google you forgot to add the `bold`/`700` weight.

---

For the `img`—do we want to stretch it all the way across the element it’s inside of?
Which `display` value will do that?
